src/ts/components/seekbar.ts

```diff
@@ -567,6 +591,20 @@ export class SeekBar extends Component<SeekBarConfig> {
 
     let seeking = false;
 
+    const setAriaSliderValue = (e: MouseEvent | TouchEvent) => {
+      let currentSeekValue;
+
+      if (this.sliderType === 'live') {
+        currentSeekValue = Math.floor(this.getOffset(e) * this.player.getMaxTimeShift());
+      } else if (this.sliderType === 'vod') {
+        currentSeekValue = StringUtils.secondsToTime(Math.floor(this.getOffset(e) * this.player.getDuration()));
+      } else {
+        currentSeekValue = Math.floor(this.getOffset(e) * 100);
+      }
+
+      this.seekBar.attr('aria-valuenow', currentSeekValue.toString());
```

Comment:
`aria-valuenow` seems od for a seekbar 🤔

Reply:
Seekbar is a slider in a sense and for web accessibility purposes we need to show the user what is the current value he is on